### ISSUE TITLE
Vue loading error

### ISSUE THREAD

**Issue by gqhuang14920** [chanind/hanzi-writer#227]
![image](https://user-images.githubusercontent.com/20200142/113509869-67892a00-958a-11eb-93d6-56b4e6d4251d.png)

**Comment by chanind** [comment]
which version of node and hanzi-writer is this?

**Comment by vidsai** [comment]
我也遇到这个问题了，我的是"3.1.0"版本

**Comment by vidsai** [comment]
> which version of node and hanzi-writer is this?

I also encountered this problem, my version is "3.1.0"，node(v8.14.0)

**Comment by chanind** [comment]
Could you try a more recent version of node? It might be that older node versions don't work with some of the newer JS syntax

**Comment by vidsai** [comment]
> Could you try a more recent version of node? It might be that older node versions don't work with some of the newer JS syntax

Yes, the version is too low, just upgrade to a higher version, thank you
